wandb/run_manager.py

```diff
@@ -98,6 +98,11 @@ def on_created(self):
     def on_modified(self):
         pass
 
+    def on_renamed(self, new_path, new_name):
+        self.file_path = new_path
+        self.save_name = new_name
+        pass
```

Comment:
Does pass give us anything here?

Reply:
nope. that was leftover from when I first wrote it empty